### DISCUSSION TITLE
New Modifier Requirement Question

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#138] [Q&A] [answered]
Good Afternoon All. I wanted to see if I could get clarification on the Modifier requirement for 2025. Per the 45 CFR 180.(b)(2)(iv)(c) "Beginning January 1, 2025, any modifier(s) that may change the standard charge that corresponds to a hospital item or service, including a description of the modifier and how it changes the standard charge."

1. Does this only apply to CDM charge lines built out (hard coded)? Meaning if the modifier is built into the CDM. OR do you also need to include the soft coded modifiers as well?

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question. 

In in the CY 2024 OPPS/ASC Final Rule, we indicated hospitals would be required to include modifiers only when they are necessary to provide the additional context needed for the standard charges the hospital has established. We agreed it is unnecessary to include modifiers that do not impact or change the standard charges established by the hospital. We clarified that a hospital would not be required to encode all combinations of codes, including modifiers, for each standard charge established. Instead, the hospital would be required to separately encode the modifiers and indicate what effect the modifier would have on the standard charge established by the hospital when used in combination with a procedure or service. For example, a hospital's contract with a third-party payer may indicate that when the service(s) provided by the hospital are greater than that usually required for the listed procedure, the hospital may identify this by adding modifier ‘22' to the usual procedure number and the payer will increase the allowed amount for the procedure by 125 percent of the 5-digit procedure code `allowable'. To reduce burden, the hospital would encode the standard charge associated with each 5-digit code, as they have been established, and then separately encode each modifier that may change the standard charge by including a description of the modifier and the way it modifies the standard charge. Please refer to the discussion starting at [88 FR 82104](https://www.federalregister.gov/d/2023-24293/p-5023).

Instructions on how to encode modifiers in the required template or schema is available on the [CMS Hospital Price Transparency Data Dictionary GitHub](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation) site.  Examples of how to encode modifiers in the template can be found [here](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples).
